File: Beecrowd/C/1009.c
#include <stdio.h>

int main() {
    char nomeVendedor[50];
    double salarioFixo, vendasEfetuadas;
    fgets(nomeVendedor, sizeof(nomeVendedor), stdin);
    scanf("%lf%lf", &salarioFixo, &vendasEfetuadas);

    double salarioTotal = salarioFixo + 0.15*vendasEfetuadas;
    printf("TOTAL = R$ %.2lf\n", salarioTotal);

    return 0;
}

File: Beecrowd/JavaScript/1009.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const vendedor = lines.shift();
const salarioFixo = parseInt(lines.shift());
const valorVendido = parseFloat(lines.shift());

const salarioFinal = salarioFixo + 0.15*valorVendido;
console.log(`TOTAL = R$ ${salarioFinal.toFixed(2)}`);
